### ISSUE TITLE
training strategy

### ISSUE THREAD

**Issue by pimaomao** [SeongjinPark/RDFNet#1]
Hi, thanks for your code. I try to reproduce your work by Pytorch on NYUD dataset. But only got MeanIOU: 0.435 using single scale evaluation. It improves the accuracy of RGBonly RefineNet slightly, far from your result reported in your paper.  Could you kindly give me some advice or is there some special training detail?

**Comment by SeongjinPark** [comment]
Hi,
I'm really sorry for late reply. I've been busy preparing my doctoral
dissertation.
Even though almost all the training details are provided in the paper, I'd
like to suggest a few checklists as follows:
1. In the dataset, images with 640x480 resolution should be used for
training rather than cropped 560x425 images. (white boundaries should be
shaved from 640x480 images)
2. The batchnorm layer should be checked carefully whether it works in the
same manner with the Caffe because the layer is often implemented
differently depending on libraries.
3. Data augmentation should be properly applied.
4. Check the location of drop-out layers and the learning rate (multiplied
by 0.1 to base lr) for resnet feature layers.

I hope it would be helpful for your work.
Thanks for your interest.

2018-05-30 11:24 GMT+09:00 pimaomao <notifications@github.com>:

**Comment by pimaomao** [comment]
Hi,

Thanks for your kind advise. It is much helpful for me. I will follow your suggestion to try again.
I hope everything goes well with your doctoral dissertation.

Best regards,

Hanchao jia  


----- Original Message -----
